src/main/scala/TwitterSetupAndLog.scala

```diff
@@ -0,0 +1,42 @@
+import java.util.regex.Pattern
+import org.apache.log4j.{Level, Logger}
+import com.typesafe.config.ConfigFactory
+import scala.io.Source
+import java.io.InputStream
+
+
+object TwitterSetupAndLog {
```

Comment:
the object is not coherent/takes multiple responsibilities which is a bad design practice.
check this out: 
* https://team-coder.com/solid-principles/
* https://thebojan.ninja/2015/04/08/high-cohesion-loose-coupling/